test/logger.test.js

```diff
@@ -998,6 +998,65 @@ describe('Should support child loggers & defaultMeta', () => {
     });
   });
 
+  it('child default meta takes precedence over parent default meta', (done) => {
+    const assertFn = ((msg) => {
+      assume(msg.level).equals('info');
+      assume(msg.message).equals('dummy message');
+      assume(msg.service).equals('audit-service');
+      done();
+    });
+
+    const logger = winston.createLogger({
+      defaultMeta: { service: 'user-service' },
+      transports: [
+        mockTransport.createMockTransport(assertFn)
+      ]
+    });
+
+    const childLogger = logger.child({ service: 'audit-service' })
+    childLogger.info('dummy message')
+  });
+
+  it('child log meta takes precedence over both child & parent default meta', (done) => {
+    const assertFn = ((msg) => {
+      assume(msg.level).equals('info');
+      assume(msg.message).equals('dummy message');
+      assume(msg.service).equals('child-service');
+      done();
+    });
+
+    const logger = winston.createLogger({
+      defaultMeta: { service: 'user-service' },
+      transports: [
+        mockTransport.createMockTransport(assertFn)
+      ]
+    });
+
+    const childLogger = logger.child({ service: 'audit-service' });
+    childLogger.info('dummy message', { service: 'child-service'});
+  });
+
+
+  it('child log meta takes precedence over both child & parent default meta even if not present in defaults', (done) => {
+    const assertFn = ((msg) => {
+      assume(msg.level).equals('info');
+      assume(msg.message).equals('dummy message');
+      assume(msg.service).equals('child-service');
+      assume(msg.requestId).equals('451');
+      done();
+    });
+
+    const logger = winston.createLogger({
+      defaultMeta: { service: 'user-service' },
+      transports: [
+        mockTransport.createMockTransport(assertFn)
+      ]
+    });
+
+    const childLogger = logger.child({ service: 'audit-service' });
+    childLogger.info('dummy message', { service: 'child-service', requestId: '451'});
```

Comment:
Let's definitely make sure to have tests on `.log` and not just `.info` so we can get consistency this time : )

Reply:
That's a very good point. Especially after my conversation in regards to @kbirger's issues.